packages/strapi-provider-upload-cloudinary/lib/index.js

```diff
@@ -16,8 +16,17 @@ module.exports = {
     return {
       upload(file, customConfig = {}) {
         return new Promise((resolve, reject) => {
+          const config = {
+            resource_type: 'auto',
+            public_id: file.hash,
+          };
+
+          if (file.ext) {
+            config.format = file.ext.replace(/^\./, ''); // remove leading dot
```

Comment:
Looks like this is documented to be a field to specify conversion on cloudinary. I feel like this is going to do some processing we might avoid in most cases 🤔